Ports/serenity-theming/package.sh

```diff
@@ -21,7 +21,8 @@ post_install() {
     cp -r "${workdir}/fonts" "${SERENITY_INSTALL_ROOT}/res/"
     cp -r "${workdir}/color-palettes" "${SERENITY_INSTALL_ROOT}/res/"
     cp -r "${workdir}/emoji" "${SERENITY_INSTALL_ROOT}/res/"
-    cp -r "${workdir}/terminal-colors" "${SERENITY_INSTALL_ROOT}/res/"
+    # FIXME: Update this path upstream.
```

Comment:
Feel free to also make a PR on the theming repo with this change (with a link to this PR)